lib/commands/project.ts

```diff
@@ -88,38 +206,90 @@ export class CreateNativeCommand extends ProjectCommandBase {
 }
 $injector.registerCommand("create|native", CreateNativeCommand);
 
-export class InitHybridCommand extends ProjectCommandBase {
+export class InitCommand extends InitProjectCommandBase {
+	constructor($project: Project.IProject,
+		$errors: IErrors,
+		$fs: IFileSystem,
+		$logger: ILogger) {
+		super($project, $errors, $fs, $logger);
+	}
+
+	public execute(args: string[]): IFuture<void> {
+		return (() => {
+			if(this.isCordovaProject().wait()) {
+				this.$logger.info("Trying to create hybrid project.");
+				this.createProjectFileFromExistingProject(projectTypes.Cordova).wait();
+			} else if(this.isNativeProject().wait()) {
+				this.$logger.info("Trying to create native project.");
```

Comment:
Attempting to initialize NativeScript project.


-> This kind of votes for my Apache Cordova comment from above :D